python/salpytools/salpylib.py

```diff
@@ -440,23 +444,50 @@ def getCurrentEvent(self):
     def getCurrentCommand(self):
         return self.getCurrent()
 
-    def waitEvent(self, tsleep=None, timeout=None):
-        """ Loop for waiting for new event """
+    def waitEvent(self, tsleep=None, timeout=None, after_timeStamp=-1):
+        """
+        Loops while waiting for a new event.
+
+        This function waits until the class gets a new event via the
+        variable self.newEvent. The self.newEvent is crontrolled by the run_Event function.
+        - Once the self.newEvent changes to True, the function makes sure that the timeStamp
```

Comment:
Delete the first "the"; the second is OK.

Reply:
fixed